core/src/main/scala/kafka/common/ZkNodeChangeNotificationListener.scala

```diff
@@ -95,7 +95,7 @@ class ZkNodeChangeNotificationListener(private val zkUtils: ZkUtils,
             if (data != null) {
               notificationHandler.processNotification(data)
             } else {
-              logger.warn(s"read null data from $changeZnode when processing notification $notification")
+              warn(s"read null data from $changeZnode when processing notification $notification")
```

Comment:
Just a quick question, why do we want to make these changes? I think we want to go in the other direction and change the existing usages to use `logger` (but in a separate PR).

Reply:
Ok, then I think I misunderstood you. So I thought that we need the Logging class as we have a few methods (logIdent, swallowing) that we use in many cases.
How do you imagine doing the migration for scala-logging? Shall we just directly mix classes with LazyLogging and throw away the Logging class? Is it good if I just use string interpolation in place to make the equivalent of `logIdent`? Do I need a KIP or that refactor?
I agree it can be a separate PR (and then this can be closed).

Reply:
I was thinking that we'd replace `logIdent` by something similar to `org.apache.kafka.common.utils.LogContext`, but I had not thought it through. So, your changes here are fine until we figure out a good strategy for replacing `logIdent` while taking advantage of `scala-logging`.

One possibility would be to check if `scala-logging` would consider supporting  prefix functionality.

Reply:
I see. I created a JIRA for the follow-on work (KAFKA-6187) and I'd intend to progress with that. Shall I revert these changes where I remove the "logger." (if tis prevents this PR going in)?

Reply:
Thanks. You can keep it as it is in this PR.